packages/effects-core/src/math/value-getters/value-getter-map.ts

```
@@ -0,0 +1,93 @@
+    return new BezierCurveQuat(props);
+  },
+  [spec.ValueType.COLOR_CURVE] (props: spec.ColorCurveData) {
+    return new ColorCurve(props);
+  },
+  [spec.ValueType.VECTOR4_CURVE] (props: spec.Vector4CurveData) {
+    return new Vector4Curve(props);
+  },
+};
+
+export function createValueGetter (args: any): ValueGetter<any> {
+  if (!args || !isNaN(+args)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)